integration-tests/actions/ocr2vrf_actions/ocr2vrf_steps.go

```diff
@@ -0,0 +1,210 @@
+package ocr2vrf_actions
+
+import (
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	ocr2vrftypes "github.com/smartcontractkit/ocr2vrf/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/core/services/keystore/chaintype"
+	chainlinkutils "github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+)
+
+func SetAndWaitForVRFBeaconProcessToFinish(t *testing.T, ocr2VRFPluginConfig *OCR2VRFPluginConfig, err error, vrfBeacon contracts.VRFBeacon) {
```

Comment:
why err is sent as parameter to this method?

Reply:
automatic mistake when I used "wrap into method" shortcut in IDE. nice catch, thanks!

Reply:
fixed